aioamqp/tests/test_publish.py

```diff
@@ -23,6 +23,20 @@ def test_publish(self):
         self.assertIn("q", queues)
         self.assertEqual(1, queues["q"]['messages'])
 
+    @testing.coroutine
+    def test_empty_publish(self):
+        # declare
+        yield from self.channel.queue_declare("q", exclusive=True, no_wait=False)
+        yield from self.channel.exchange_declare("e", "fanout")
+        yield from self.channel.queue_bind("q", "e", routing_key='')
+
+        # publish
+        yield from self.channel.publish("", "e", routing_key='')
+
+        queues = self.list_queues()
+        self.assertIn("q", queues)
+        self.assertEqual(1, queues["q"]["messages"])
```

Comment:
we might want to test the content here ?

Reply:
Imho this test should only use the publish method from this library and not depends on the other methods such as `basic_consume` to limit the module bonds. but that's a point of view.

The `message_bytes` should be enough ?

Reply:
By the way, a more complete end-to-end test is defined here, if you want to see an example : https://github.com/Polyconseil/aioamqp/blob/48f0638662341af2bcf0a09613ba4b7971a2a0ee/aioamqp/tests/test_consume.py#L58

Reply:
Nice to know. I've amended the commit with the `message_bytes` assertion.